Something/WebApplication/webpack.config.js

```diff
@@ -2,10 +2,18 @@
 const tsImportPluginFactory = require('ts-import-plugin')
 module.exports = {
     mode: 'development',
-    entry: './wwwroot/js/index.tsx',
+    entry: {
+        login: './wwwroot/js/apps/authentication/AuthenticationEntry.tsx'
```

Comment:
Why no create separate entry points for each page ?

Reply:
Yes, its like that only. Right now only 1 page "Authentication" hence 1 entry point.